README.md

```diff
@@ -370,6 +379,11 @@ You can define special `setup` and `teardown` functions, which run before and
 after each test case, respectively. Use these to load fixtures, set up your
 environment, and clean up when you're done.
 
+You can also define `setup_file` and `teardown_file`, which will run once per file,
+before the first and after the last test, respectively. 
+__WARNING__ these will not be run in parallel mode!
```

Comment:
Can we change "parallel mode" to "--jobs mode" or similar?

What does this mean for users? Could we add wording on the `--jobs` help text?

Reply:
Good catch, that one is outdated, they also do work with parallel mode now.